projects/jdwp/serializers/serializer_codegen.py

```diff
@@ -0,0 +1,99 @@
+import re
+import os
+from projects.jdwp.defs.schema import Type, Struct, CommandSet, Command, Field
+from projects.jdwp.defs.command_sets.reference_type import ReferenceType
+
+
+class CommandSerializerGenerator:
+    def __init__(self):
+        self.serializer_code: str = ""
+
+    def generate_field_serializer(self, field: Field) -> str:
+        serializer_code: str = ""
+        if isinstance(field, Struct):
+            for subfield in field.fields:
+                serializer_code += self.generate_field_serializer(subfield)
+        elif field.type == Type.INT:
+            serializer_code = (
+                f"        serialized_data += command.{field.name}.to_bytes(4, 'big')"
+            )
+        elif field.type == Type.STRING:
+            serializer_code = (
+                f"        serialized_data += command.{field.name}.encode('utf-8')"
```

Comment:
This is not the format that we want. The spec says:
> A UTF-8 encoded string, not zero terminated, preceded by a four-byte integer length.
I think it will be a good approach to introduce some kind of input and output stream abstractions that knows how to read/write values of all types represented by `PrimitiveType` union.